pkg/controller/workload/kubernetes/scheduler/scheduler.go

```diff
@@ -105,7 +105,7 @@ func UpdatePredicate(e event.UpdateEvent) bool {
 
 // Setup adds a controller that schedules KubernetesApplications.
 func Setup(mgr ctrl.Manager, l logging.Logger) error {
-	name := "scheduler/" + strings.ToLower(workloadv1alpha1.KubernetesApplicationKind)
+	name := strings.ToLower(workloadv1alpha1.KubernetesApplicationGroupKind)
```

Comment:
Does this not conflict with the KubernetesApplication controller name above?

Reply:
Ah - the removal of the prefixes like `scheduler/` in this PR was unintentional. Will fix.

Reply:
Fixed - PTAL.